ui/components/app/name/name-details/name-display.tsx

```diff
@@ -36,31 +44,62 @@ const NameDisplay = memo(
 
     const hasDisplayName = doesHaveDisplayName(name);
 
+    // Get icon properties from utility function
+    const iconProps = trustSignalDisplayState
+      ? getTrustSignalIcon(trustSignalDisplayState)
+      : null;
```

Comment:
This is the reason why we are getting all the test snapshots updated. trustSignalDisplayState is always set. This will always return the default icon. We should change trustSignalDisplayState being always set.